packages/epo-react-lib/src/atomic/Button/Button.tsx

```diff
@@ -0,0 +1,59 @@
+import { forwardRef, ReactNode, HTMLProps } from "react";
+import { IconKey } from "@/svg/icons";
+import IconComposer from "@/svg/IconComposer";
+import * as Styled from "./styles";
+
+export type ButtonStyleAs = "primary" | "secondary" | "tertiary" | "educator";
+
+interface ButtonProps extends HTMLProps<HTMLButtonElement> {
+  children: ReactNode;
+  className?: string;
+  icon?: IconKey;
+  iconSize?: number;
+  isBlock?: boolean;
+  styleAs?: ButtonStyleAs;
+  /** This is a disabled style without disabling the button.
+   * Good for a11y - button is visible even if form isn't complete.
+   */
+  isInactive?: boolean;
+}
+
+const Button = forwardRef<HTMLButtonElement, ButtonProps>(
+  (
+    {
+      children,
+      icon,
+      iconSize,
+      isBlock,
+      styleAs,
+      isInactive,
+      className,
+      ...buttonProps
+    },
+    ref
+  ) => (
+    <Styled.Button
+      $isBlock={isBlock}
+      $styleAs={styleAs}
+      $hasIcon={!!icon}
+      aria-disabled={isInactive || undefined}
+      data-testid="button"
+      {...{ ...(buttonProps as any), className, ref }}
+    >
+      <>
+        {icon && (
+          <IconComposer
+            icon={icon}
+            size={iconSize}
+            aria-hidden={children && true}
```

Comment:
this was previously set as `role='presentation'` if children were present. Changed this to `aria-hidden` to follow recommendations not to change the implicit role of SVG's and that `role='presentation'` [does not actually hide the element from screenreaders](role='presentation')